src/jit/jitconfigvalues.h

```diff
@@ -200,8 +200,13 @@ CONFIG_INTEGER(AltJitAssertOnNYI, W("AltJitAssertOnNYI"), 0) // Controls the Alt
 CONFIG_INTEGER(AltJitAssertOnNYI, W("AltJitAssertOnNYI"), 1) // Controls the AltJit behavior of NYI stuff
 #endif                                                       // defined(_TARGET_ARM64_) || defined(_TARGET_X86_)
 
+#if defined(_TARGET_X86_) || defined(_TARGET_AMD64_)
+CONFIG_INTEGER(EnableSSE3_4, W("EnableSSE3_4"), 1) // Enable SSE3, SSSE3, SSE 4.1 and 4.2 instruction set as default
```

Comment:
Using _TARGET_XARCH_ in this header file leads to compilation error which I have not pursued further.

---
In reply to: [89866312](https://github.com/dotnet/coreclr/pull/8291#discussion_r89866312) [](ancestors = 89866312)